doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools component descriptions
+=================================================
+
+The Plutus team has provided a set of “Tools”---components, libraries and 
```

Comment:
When viewed through the rendered html pages, the three dashes become a longer dash, like an em dash versus a hyphen. Do you typically view the html rendered version on Github? I just recently learned that you can do that. In case you don't already know this, try going to the bottom of this page. The click the link "Show all checks." At the bottom of that list (you may need to scroll down in the window), select the "Details" link that appears to the right of "docs/readthedocs.org:plutus-apps — Read the Docs build succeeded!" Then you can view the html rendered version like how it is meant to be seen.

Reply:
Reminder that you can view the rendered html pages with this link: 
https://plutus-apps--711.org.readthedocs.build/en/711/

Also, we have removed the "Dependencies" and the "State" information from this page.